hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/schedule/ScheduleExecutionSpecs.java

```diff
@@ -2064,7 +2091,7 @@ private HapiSpec scheduledFreezeWorksAsExpected() {
         return defaultHapiSpec("ScheduledFreezeWorksAsExpected")
                 .given(
                         cryptoCreate(PAYING_ACCOUNT),
-                        overriding(SCHEDULING_WHITELIST, FREEZE),
+                        overriding(SCHEDULING_WHITELIST, "Freeze"),
```

Comment:
Is this a readability change? Curious why this would be an issue. I see a few of these changes in this file

Reply:
Mostly readability, yes.